src/recode.jl

```diff
@@ -176,7 +178,7 @@ function recode!(dest::CategoricalArray{T}, src::CategoricalArray, default::Any,
     # For missing values (0 if no missing in pairs' keys)
     indexmap[1] = 0
     for p in pairs
-        if ismissing(p.first)
+        if any(ismissing.(p.first))
```

Comment:
Better use `any(ismissing, p.first)`.

Reply:
agreed, fixed

Reply:
@nalimilan Now I remember why I did not used your code :). In the last commit I have rewritten all tests of `p.fits` in a cleaner and consistent manner handling both vectors and atoms in `p.first`.